pkg/gencmd/providers.go

```diff
@@ -0,0 +1,22 @@
+package gencmd
+
+import (
+	"net/http"
+
+	"github.com/google/go-cloud/wire"
+
+	"github.com/izumin5210/grapi/pkg/grapicmd"
+)
+
+func ProvideGrapiCtx(ctx *Ctx) *grapicmd.Ctx         { return ctx.Ctx }
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideGrapiCtx should have comment or be unexported